tools/debian/control

```diff
@@ -70,26 +74,26 @@ Description: Cockpit deployment and developer guide
  or extend Cockpit.
 
 Package: cockpit-docker
-Architecture: amd64 armel armhf i386
+Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python
```

Comment:
Not sure which Debian package you're referring to, but I definitely see this installed when I use our http://cockpit-project.org/running.html packages on Debian:

```
root@unassigned-hostname:~# grep VERSION /etc/os-release 
VERSION_ID="8"
VERSION="8 (jessie)"
root@unassigned-hostname:~# zgrep python /usr/share/cockpit/docker/* 
/usr/share/cockpit/docker/docker.min.js.gz:                l.hidden || i ? l.hidden && i && (i.close("closed"), i = null) : i = l.spawn([ "python", "-c", u, "monitor" ], {
/usr/share/cockpit/docker/docker.min.js.gz:                var r = l.spawn([ "python", "-c", u, s ].concat(i), {
/usr/share/cockpit/docker/docker.min.js.gz:                var n = l.spawn([ "python", "-c", u, "reset-and-reduce" ], {
```